workflows/scRNAseq/standard-scanpy/README.md

```diff
@@ -0,0 +1,23 @@
+# Standard scRNA-seq Workflow using Scanpy and Anndata
```

Comment:
What is even standard. Is clustering the thing you do here ?

Reply:
Preprocessing and clustering. I will rename it accordingly.